Cargo.toml

```diff
@@ -185,7 +187,7 @@ path = "./svd2utra"
 # path = "./api/xous-api-names"
 # [patch.crates-io.xous-api-susres]
 # path = "./api/xous-api-susres"
-# [patch.crates-io.xous-api-log]
-# path = "./api/xous-api-log"
+[patch.crates-io.xous-api-log]
```

Comment:
@bunnie how would you like to handle patches to crates such as `xous-api-log`?

Reply:
I think for this one we'd just publish the change. i looked at it and it's fairly minor and cleaning up some of the cobwebs that were in that crate anyways.